src/index.js

```diff
@@ -69,16 +79,19 @@ var sleepRepository;
 var activityRepository;
 var user;
 
-Promise.all([userData]).then(element => {
-  userData = element[0]
+Promise.all([userData, sleepData, hydrationData, activityData]).then(element => {
+  userData = element[0];
+  sleepData = element[1];
+  hydrationData = element[2];
+  activityData = element[3];
   randomId = Math.floor(Math.random() * (50 - 1) + 1);
   userRepository = new UserRepository(userData, randomId);
-  hydrationRepository = new HydrationRepository(hydrationData, randomId);
   sleepRepository = new SleepRepository(sleepData, randomId);
+  hydrationRepository = new HydrationRepository(hydrationData, randomId);
   activityRepository = new ActivityRepository(randomId, activityData);
```

Comment:
It may be worth making activityData the first parameter or argument in order to match the other repos.